pkgs/tools/misc/zoxide/default.nix

```diff
@@ -1,28 +1,31 @@
-{ stdenv
+{ lib
 , fetchFromGitHub
 , rustPlatform
-, withFzf ? true, fzf
+, withFzf ? true
+, fzf
 }:
-
-rustPlatform.buildRustPackage rec {
```

Comment:
just curious, why do you prefer `let` over `rec`?

Reply:
I'm trying to stay away from `rec` as much as possible because I think it's cleaner. I seem to remember that I had (or read) a conversation about `rec` vs `let` on IRC a while back (I think it was with @adisbladis. If not, sorry for the ping), but I can't find it in my logs (it was probably in April or prior to the end of May). I believe it boiled down to `rec`-less being faster to evaluate?